app/controllers/trading_partners_controller.rb

```diff
@@ -16,13 +16,19 @@ def update
       redirect_to goods_nomenclature_path(
         country: @trading_partner.country,
       )
+    elsif should_not_render_errors?
```

Comment:
the "should" part sounds like a test,
what about
`does_not_render_errors` or `not_render_errors` or just `or `hide_errors` (<- my favourite)